collector/pg_stat_database.go

```diff
@@ -280,152 +267,218 @@ func (PGStatDatabaseCollector) Update(ctx context.Context, instance *instance, c
 		if err != nil {
 			return err
 		}
+		datidLabel := "unknown"
+		if datid.Valid {
+			datidLabel = datid.String
+		}
+		datnameLabel := "unknown"
+		if datname.Valid {
+			datnameLabel = datname.String
+		}
 
+		var numBackendsMetric float64
```

Comment:
This should probably be reset to 0 to avoid row-torow leaking int the for loop.

```suggestion
		numBackendsMetric := 0.0
```